### PR TITLE
📝 Add TA roadmap page

### PR DESCRIPTION
**Pull request by olyism**
Add a simple Test Analytics roadmap page.

image

### CODE REVIEW

**Review by plaindocs — APPROVED** [review]
This works for me. Can we set up a recurring Linear ticket to update it every X weeks @meiqimichelle ?

**Review by karensawrey — CHANGES_REQUESTED** [review]
Great job! A few improvements in the review. I'm pretty sure, @plaindocs will be able to make some things even more breve and clear after these are imlemented.

File: pages/test_analytics/roadmap.md.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Now
+We're working on these items now, and will have updates soon:
+
+- **Search and filter**  
+  Quickly find a given test based on its title or text contents
+- **Additional test collectors**
+  We'll be publishing additional collectors to bring full support for Android, Python and .NET to Test Analytics
+
+## Next
+We've got a pretty good idea of the work required for these items, and we'll be prototyping these soon:
```
[the remark this answers is not in the document]
Comment (olyism) [reply pages/test_analytics/roadmap.md.erb:16]:
```suggestion
We've got plans for many exciting new things, and these are the ones we'll be prototyping soon:
```

File: pages/test_analytics/roadmap.md.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- **Action your analytics**  
+  Tooling to support teams who work on test suites: assign ownership, alert on changes, and track progress towards issue resolution.
+- **Set goals**  
+  Set goals for test suite reliability or performance over time, and trigger actions when thresholds are met.
+
+## Later
+These are areas of interest for us — we're still collecting feedback and suggestions to help shape our implementation:
+
+- Deeper integration with Buildkite agents and Pipelines
+- More ways to organise and group your tests — by namespace, filename or team, for example — so you can quickly find the patterns and data you need
```
[the remark this answers is not in the document]
Comment (olyism) [reply pages/test_analytics/roadmap.md.erb:31]:
```suggestion
- More ways to organize and group your tests — by namespace, filename, or team. For example — to quickly find the patterns and data you need.
```

**Comment by olyism** [comment]
hi @meiqimichelle I've added some very minor text suggestions. Passing the ball over to you ⚽️ to take a look to commit the suggestions or make other changes.

**Comment by karensawrey** [comment]
(CC: @olyism) @meiqimichelle I'm changing my review status to Approved - please look at my suggestions & merge the ones you agree with. Then let's get this rollin'!

**Review by karensawrey — APPROVED** [review]
Do merge after committing the suggested changes.

**Comment by meiqimichelle** [comment]
@olyism I've pulled this down to take a look -- apologies it's taken me so long to confirm. I'll push a few minor edits up while I'm here, but I want to confirm this list with the team this week before making public. Merging won't push out longer than end of this week or early next, I'm sure of it!

**Comment by meiqimichelle** [comment]
Also, I can't figure out why this particular dot point isn't parsing the line break/paragraph break. Womp womp.
Screen Shot 2022-06-28 at 9 21 27 PM

**Comment by plaindocs** [comment]
@meiqimichelle you **do** want the line break? or you don't? Either way, you're doing stuff that markdown doesn't really encourage. There is no concept of definition list. 

I've fixed it for what I think you want.

**Comment by olyism** [comment]
Too quick @plaindocs ⚡️ I was looking at the same thing. @meiqimichelle you can use double space for line breaks in bullet points.

No worries about the time. I was just checking in :) Drop me a line when it's ready to go and I can help you rebase the branch to pull in the recent nav changes.

**Comment by meiqimichelle** [comment]
(I don't have a strong opinion about the line break -- I was just trying to get it to match the others in the design)

**Comment by karensawrey** [comment]
@meiqimichelle @olyism where are we with this one?

**Comment by plaindocs** [comment]
@olyism what do you think about having a distinct page layout for roadmaps, to highlight the fact that these aren't "solid docs"?

**Comment by olyism** [comment]
hi @plaindocs yes, keen to have a different page layout for Roadmaps. I've got a backlog ticket to provide a different design for it, which I haven't had a chance to start yet.

Question to everyone — are we okay to scope this PR to use the standard template for the release? If we're waiting for the design we might be delayed a bit.

**Comment by plaindocs** [comment]
There doesn't appear to be much action here, but regardless I'm happy to merge this first and iterate on the design after.

**Comment by olyism** [comment]
hi @meiqimichelle I'm pruning the PRs and issue in Docs' GitHub repo. Can you confirm if we still want to have TA roadmap on Docs, or can we de-prioritise and cancel this PR and look into next year when we start to look at a proper roadmap solution.

**Comment by meiqimichelle** [comment]
Uff. This PR is quite stale at this point, hey? I can always re-create, if needed. Feel free to close this one for now.

**Comment by olyism** [comment]
Thanks @meiqimichelle :) closing this PR for now. Let's chat about public-facing roadmaps when we reboot it in the next quarter.

**Pull request closed without merging**